client-app/shared/catalog/components/product-card.vue

```diff
@@ -196,3 +285,45 @@ const variationsCount = computed(() => {
   return result;
 });
 </script>
+
+<style scoped lang="scss">
+.product-card {
+  &__variations-button {
+    @apply hidden;
+
+    &--list {
+      @container (min-width: theme("containers.xl")) {
+        @apply block;
+      }
+    }
+  }
+
+  &__variations-link-button {
+    @apply block;
+
+    &--list {
+      @container (min-width: theme("containers.xl")) {
+        @apply hidden;
+      }
+    }
+  }
+
+  &__variants-wrapper {
+    @apply border-t border-neutral-200 p-6 pt-4 hidden;
+
+    &--list {
```

Comment:
better to use mod `&--hidden` instead of `&--list`

Reply:
I moved the modifier to the root element, so now it doesn't make sense to have a modifier on this button